api/contrib/envoy/extensions/filters/network/generic_proxy/v3/route.proto

```diff
@@ -68,5 +94,11 @@ message RouteConfiguration {
   string name = 1 [(validate.rules).string = {min_len: 1}];
 
   // The match tree to use when resolving route actions for incoming requests.
-  xds.type.matcher.v3.Matcher routes = 2 [(validate.rules).message = {required: true}];
```

Comment:
I guess we needn't support backward-compatibility in the generic proxy filter :)